src/xFactor/Competition.java

```diff
@@ -0,0 +1,44 @@
+package xFactor;
+
+import java.util.ArrayList;
+
+public abstract class Competition {
+	private static ArrayList<Judge> judges = new ArrayList<Judge>();

```

Comment:
Do not use static fields for class properties

Reply:
They need to be the same for every instance of Competition. I made them all non-static and the data was overridden every time I created an instance in order to use them.